include/utils.php

```diff
@@ -1768,8 +1768,7 @@ function sugar_die($error_message, $exit_code = 1)
 {
     global $focus;
     sugar_cleanup();
-    //echo $error_message;
-    //die($exit_code);
+    echo $error_message;
```

Comment:
Why was this removed?

Reply:
I didn't remove this. It must have come from an a previous commit message